application/models/GoogleSheets_Model.php

```diff
@@ -108,6 +108,60 @@ public function addNewRecord($email, $fullName, $paymentMethod)
         $result = $this->service->spreadsheets_values->update($this->spreadsheetId, $range, $body, $params);
     }
 
+    /**
+     * Mark an attendees attendance in the google sheets
+     *
+     * @param string $eventName The name of the event
+     * @param string $email The email of the user to record.
+     * @param string @upi The upi of the user if it exists
+     */
+    public function markAsPresent($eventName, $email = null, $upi = null)
+    {
+        // This should probably be in a try/catch block
+        if (is_null($email) and if_null($upi)) {
+            throw new Exception("You need to enter an email or UPI.");
+            return false;
+        }
+
+        if (is_null($eventName)) {
+            throw new Exception("You need to enter a sheet name.");
+            return false;
+        }
+        
+        //  Navigate to the correct spreadsheet
+        $this->setCurrentSheetName($eventName);
+
+        //  If they do exist, find the cell coordinates where you want to place a 'P'
+        $emailCell = $this->getCellCoordinate($email, 'B');
+
+        list(, $row) = NULL;
+
+        if ($emailCell != NULL)
+            list(, $row) = $this->convertCoordinateToArray($emailCell);
+        else {
+            $upiCell = $this->getCellCoordinate($upi, 'B');
+
+            if ($upiCell != NULL)
+                list(, $row) = $this->convertCoordinateToArray($upiCell);
+            else
+                return false;
```

Comment:
Wrap all if/else bodies in `{}` braces, even if they're only one line. This is definitely just a coding standard thing but it's a good thing to get used to for PHP since PSR-12 ([PHP Standards Recommendations](https://www.php-fig.org/psr/psr-12/)) has the following in section 5.

> The body of each structure MUST be enclosed by braces. This standardizes how the structures look and reduces the likelihood of introducing errors as new lines get added to the body.